pkg/controllers/metrics/pod/controller.go

```diff
@@ -68,7 +68,7 @@ var (
 		prometheus.SummaryOpts{
 			Namespace:  "karpenter",
 			Subsystem:  metrics.PodSubsystem,
-			Name:       "startup_time_seconds",
+			Name:       "startup_duration_seconds",
```

Comment:
I'm fine to couple these in the same PR since the change is small but it might be worthwhile to decouple these into separate PRs since the changes are unrelated